### ISSUE TITLE
dedicated server uses cl_download_mapsrc without initializing it

### ISSUE THREAD

**Issue by Xylemon** [fte-team/fteqw#152]
https://sourceforge.net/p/fteqw/tickets/150/

ferneu1 wrote on 2022-06-10:

When you run FTE as a dedicated server and tell the server to load a map it does not have, it will try to use the value of `cl_download_mapsrcto` download the map. The problem is that `cl_download_mapsrc` only gets initialized after the code exits the dedicated server loop.

Things get really confusing because a cvar named `cl_download_mapsrc` gets registered because it is included in `QUAKEOVERRIDES` (see common/fs.c) and, because of that, the server will reply with a valid string if you request the value of that cvar. The problem is that the code will try to use the C variable named `cl_download_mapsrc`, and that one never gets initialized.

So, basically, anyone can crash a dedicated FTE server by simply telling it to load a map it doesn't have. Which was something I discovered when I mistakenly misspelled E1M1 as EE1M1
